engine/src/main/java/com/datatorrent/stram/codec/LogicalPlanSerializer.java

```diff
@@ -203,6 +221,201 @@ public boolean useForType(JavaType t)
     return result;
   }
 
+  /**
+   * @param flatten
+   * @param dag
+   * @return
+   */
+  public static Map<String, Object> convertToMap(LogicalPlan dag, boolean flatten)
```

Comment:
Why is there another method with the same name right above? What is the difference?